### ISSUE TITLE
Add support for nested argument files

### ISSUE THREAD

**Issue by jylind** [MarketSquare/robotframework-robocop#770]
I would like Robocop to support nested argument files similar to Robot Framework itself.

This would allow separating common settings from custom settings and, thus, minimizing the need of changing multiple files when something changes in the common settings

**Comment by bhirsz** [comment]
I managed to implement this, however I need few days to think about the design (the code is bit ugly). 

New implementation: 
- it is possible to import argument files inside argument files
- relative paths are resolved against current argument file directory (--ext-rules, --output, robot files paths and argument files paths)
- circular imports are detected

For example you can use following command:
```
robocop -A dev.txt test.robot
```
with files:
dev.txt
```
--exclude *doc*
-A base.txt
tests
```
base.txt
```
--exclude 0102
--reports all
```

The only problem is that user need to be careful where to define robot files paths - the argument file works like providing cli arguments so they need to be grouped together. In this case you can put them either to dev.txt or base.txt or by the end of robocop command, but you can't put some paths to one argument file and some to the other.

**Comment by bhirsz** [comment]
@jylind 

If you want, I can release alpha version of the robocop so you can test it to see if it works as expected for you

**Comment by jylind** [comment]
Yes, please, I could do that

**Comment by bhirsz** [comment]
I have release alpha version containing this change, you can install it by running:

```
pip install robotframework-robocop==2.8.0a1
```
or
```
pip install -U --pre robotframework-robocop
```

**Comment by jylind** [comment]
Hi, I managed to do some quick testing.
For most parts it worked as expected but there was one thing though.

Let's say that I have the files as in your example: `dev.txt` & `base.txt`.

For some reason if I declare my custom rules in `dev.txt` like this:
```
--ext-rules rflinter.robocop.naming
```

it works as expected

But if I declare it in the `base.txt`  I get the following error:
```
Error: Fatal error: Failed to load external rules from file "config\robocop\rflinter.robocop.naming". Verify if the file exists
```

**Comment by bhirsz** [comment]
Oh, it was my bad - I reversed the condition (I have the code that tries to resolve any relative paths, and for --ext-rules it should try to resolve relative paths but if the file doesn't exist, it should return it as it is since it could be imported package). I fixed it and included some tests for it to catch this kind of issues in the future. Thanks
